main.nf

```diff
@@ -123,10 +154,18 @@ process updateDataTable {
    stdout into outUpdateDataTable
 
    script:
-   """
-   cd /root/scripts/
-   python update_data_table.py -p /root/scripts/config.json -m $comment primary
-   """
+   if (params.production) {
+      """
+      cd /root/scripts/
+      python update_data_table.py -p /root/scripts/config.json -m $comment primary
+      """
+   }
+   else {
+      """
+      cd /root/scripts/
+      python update_data_table.py -p /root/scripts/config.json -m $comment -d True primary
```

Comment:
@rxu17 can you speak to why `-d True` is necessary here?  Is it because of the "primary" str

Reply:
I'm doing some testing around, but for some reason in the end I need to add quotes and change this line to:
`python update_data_table.py -p /root/scripts/config.json -m "$comment" primary -d`

because at least in my ec2 instance, it's not interpreting $comment as a multi-word string but as separate arguments so `table_type` is set to `public` which isn't allowed. Not sure if this will successfully run itself all the way through but it stopped throwing the error, and the command line call looks okay now based on the debug statements.

Reply:
Thanks for the explanation!  May want to add a comment about that - string arguments can be a headache.  This PR is looking great.